core/controllers/home_test.py

```diff
@@ -26,10 +26,7 @@ class HomePageTest(test_utils.GenericTestBase):
     def test_logged_out_homepage(self):
         """Test the logged-out version of the home page."""
         response = self.testapp.get('/')
-        self.assertEqual(response.status_int, 200)
-        response.mustcontain(
-            'Your personal tutor',
-            'Oppia - Gallery', 'About', 'Sign in', no=['Logout'])
+        self.assertEqual(response.status_int, 302)
```

Comment:
For this one, you do need to check that the destination location is correct, and then navigate to it (response.follow()? I'm not sure, you'll need to look up the webtest docs) and then check that it has the right terms. Don't just delete the original test.